### ISSUE TITLE
Duplicate h1 - homepage and speaking

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#82]
◦	The Able Player videos (homepage only) has a built-in off-screen H1, “Media Player”. Makes for two H1s on the page

**Comment by vavroom** [comment]
Cannot replicate. Ideally heading structure should be spot on. Realistically, as long as headings are there it's useful. Since I can't replicate, and the text `Media Player` appears to me as a `<h3>`, I'm closing.
